### ISSUE TITLE
Error in Best Servlet Practise project

### ISSUE THREAD

**Issue by chaudharisuresh997** [benbai123/JSP_Servlet_Practice#2]
Cant not resolve classs JSON Array

**Comment by chaudharisuresh997** [comment]
Project has many jars those are not included in runtime

**Comment by benbai123** [comment]
Yes, as I said at previous issue [No build management #1](https://github.com/benbai123/JSP_Servlet_Practice/issues/1), what I tried to do is providing the most basic parts excluding the whole environment.

All useful parts are the code and comments, to avoid the problem that someone probably copy files from different project and make broken environment (assume they copy jars and pom.xml from different project that include different version of jars)

For those who can solve the problem above, can also find jars themselves, but for someone who feel find jars himself difficult to solve the problem above is quite tough (probably almost impossible).
